### PR TITLE
Addressing final changes

### PR DESCRIPTION
**Pull request by excerebrose**
Fixing all changes in #1 
All changes include:
1. Addition of dockerfile & documentation for how to run a container
2. Moved API variable to environment variable
3. Moved to using 3rd part library for highlighting.
4. Misc. Cleanup

@gaieges

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM python:3.7-buster
+
+#set working directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Copy Shit over
```
Comment (gaieges) [inline Dockerfile:7]: Language plz, going to keep this repo open source :)

**Review by gaieges — APPROVED** [review]
one nit, then lets merge

File: Dockerfile (hunk 1 above)
Comment (excerebrose) [reply Dockerfile:7]: Oops.. sorry :( let me update

Comment (excerebrose) [reply Dockerfile]: @gaieges Fixed

**Review by gaieges — APPROVED** [review]
nice!

**Pull request merged**